tests/test_image_gen.py

```diff
@@ -1,39 +1,46 @@
 import hashlib
 import os
+import shutil
 import unittest
+from pathlib import Path
 
 from PIL import Image
 
 from autogpt.commands.image_gen import generate_image, generate_image_with_sd_webui
 from autogpt.config import Config
-from autogpt.workspace import path_in_workspace
+from autogpt.workspace import Workspace
 from tests.utils import requires_api_key
 
 
 def lst(txt):
-    return txt.split(":")[1].strip()
+    return Path(txt.split(":")[1].strip())
 
 
-@unittest.skipIf(os.getenv("CI"), "Skipping image generation tests")
```

Comment:
Could delete these tests or add a backlog item to fix them.

Reply:
Add the decorator I suppose

Reply:
Either way use one of the decorators from the test/utils. Either skip in ci or requires api key